dask/dataframe/io.py

```diff
@@ -462,14 +462,18 @@ def to_hdf(df, path_or_buf, key, mode='a', append=False, complevel=0,
                  "must preserve the order of its input")
 
     # handle lock default based on wether we're writing to a single entity
+    _actual_get = get if get else dask.get
```

Comment:
Ok, so that's a misunderstanding on my part. Since it's already in the code i think we might want to rush into fixing it.

Will replacing  `_actual_get` with the snippet in my previous comment work as expected?